opencog/nlp/learn/scm/link-pipeline.scm

```diff
@@ -442,6 +442,15 @@
 			(set! run-time run))))
 
 ; --------------------------------------------------------------------
+
+(define-public (observe-text plain-text)
+"
+ Wrapper to maintain backwards compatibility in NLP pipeline.
+ Passes default parameters to observe-text-mode
+"
+	(observe-text-mode plain-text "any" 24)
+)
+
 (define-public (observe-text-mode plain-text observe-mode count-reach)
```

Comment:
To avoid compiler warnings about undefined functions, its conventional to place such things after the functions they call, and not before.